### DISCUSSION TITLE
Does this support audio input from external sources?

### DISCUSSION THREAD

**Discussion by JunkMuffler** [FundamentalFrequency/.github#48] [Q&A]
I'd like to be able to hook up an external audio device like a synthesizer and record in. Is this possible, or is it a limitation of the Pi only having one audio jack?

**Comment by stonepreston** [comment]
currently no, this is not possible since the Pi does not have an audio input.

**Comment by DominusFulguris** [comment]
There are any number of USB<->Audio adapters on places like Amazon. Surely something like that could be used?

**Reply by JunkMuffler** [reply]
I think I saw in another thread that USB Audio isn't supported at this point. This would be a good solution it it were.

**Reply by stonepreston** [reply]
There would probably need to be an additional setting added to support selecting an audio input device [here](https://github.com/FundamentalFrequency/LMN-3-DAW/blob/main/Source/Modules/app_view_models/Edit/Settings/SettingsListViewModel.h). Currently only a setting for output exists.

**Comment by BlueLeef** [comment]
Were you ever able to find a solution/workaround to recording in? I just discovered the project, but the main use case I was looking for was as a sequencer. Though the self contained nature of the LMN 3 is cool too!
